### PR TITLE
build multiplatform cht-upgrade-service

### PR DESCRIPTION
**Pull request by 1yuv**
In medic/cht-core#8841, we made all cht-core images multi-platform supported. However, upgrade service was not worked on at that point. This pr makes cht-upgrade-service multi-platform and supports same multi-platform platforms supported by cht-core. 

Fixes: #36

### CODE REVIEW

**Review by mrjones-plip — APPROVED** [review]
looks good!  

since we haven't pinned the compose files to a version (they all use `cht-upgrade-service:latest`) - let's do a little coordination on when you want to merge this?  That way I can test x86 right away and you can test arm64 right away and we'll validation the new image is good.

**Comment by 1yuv** [comment]
Is there a day that we need to wait for merge? Or is it to test immediately after merge and revert if there's a problem? If so, let's merge it tomorrow.

**Comment by mrjones-plip** [comment]
Yeah - today (Tue apr 9)  is good! I'll ping you on slack to coordinate.

**Pull request merged**
